### DISCUSSION TITLE
Is it possible to download only NodeJs without npm module?

### DISCUSSION THREAD

**Discussion by obalilty** [nodejs/help#4889] [Q&A] [answered]
Hi All,
in my project i need node js but no need for npm.
Is it possible to download only NodeJs without npm module?
is NPM module is needed in order to make Node run?

Regards

**Comment by Trott** [comment, marked as answer]
You don't need npm to run Node.js, but as far as I know, we don't offer pre-built Node.js without bundling npm.  (EDIT: Except on Windows. See thread.)

**Reply by richardlau** [reply]
We do on Windows but not on other platforms.

**Reply by ghost** [reply]
@richardlau Any reason for that? Just curious.

**Reply by richardlau** [reply]
Historical? No idea, it was already like that when I first got involved (Node.js 0.10.x).

**Reply by obalilty** [reply]
@richardlau where can i find Node for windows without NPM module?

**Reply by richardlau** [reply]
From https://nodejs.org/en/download/ (or https://nodejs.org/en/download/current/) follow the "All download options" link near the bottom of the page and then either `win-x64` or `win-x86` depending on architecture.

**Reply by obalilty** [reply]
is it possible to create a package only with NodeJS? similar to Windows?

**Reply by richardlau** [reply]
There are no plans to do so.

**Reply by obalilty** [reply]
How can such a request be promoted?

**Reply by Trott** [reply]
> How can such a request be promoted?

You can open an issue with the feature request template. But before you do that, I have to ask: What problem are you trying to solve? Each Node.js release is already a huge amount of work for the volunteers who create Node.js, so adding more variants is probably going to be looked at with skepticism if there isn't a compelling use case.

**Reply by Trott** [reply]
By the way, I believe you can uninstall `npm` with `npm uninstall -g npm`. Make sure that's what you really want to do, though, obviously.

**Reply by anonymous887** [reply]
@Trott 
I wanted to become a contributor for Node.
Kindly help me getting started with it
